.aports.yml

```diff
@@ -1,4 +1,25 @@
 repos:
+  melodic-3.8:
+    main_ref: "master"
+    build_args:
+      ALPINE_VERSION: "3.8"
+      ROS_DISTRO: "melodic"
+    envs:
+      JOBS: 4
+      PURGE_OBSOLETE: "yes"
+      STACK_PROTECTOR: "off"
+      ADDITIONAL_APK_REPO: "http://alpine-ros.seqsense.org/v3.8/backports"
```

Comment:
Is this needed because the backports are not built anymore?

Reply:
Yes

Reply:
https://github.com/seqsense/aports-ros-experimental/pull/824#issue-1723131353
>  As it's too complicated to restore backports for 3.8, only ROS package build is restored.